cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/IndexedTable.java

```diff
@@ -100,7 +100,7 @@
   /**
    * Column key used to store the existence of a row in the secondary index.
    */
-  private static final byte[] IDX_COL = {'r'};
+  public static final byte[] IDX_COL = {'r'};
```

Comment:
It seems difficult to not make this public, since the migrator class is in the same package is `PartitionedFileSet`, in order to make the variables we need in `PartitionedFileSet` not public, but just protected.
If we instantiated the indexed table and did transactional writes to it, that would be one way to get round it.
Alternatively, we could at least document that it shouldn't be used?